.github/workflows/automation-check.yml

```diff
@@ -0,0 +1,27 @@
+name: Chectl
+on: [pull_request]
+
+jobs:
+  run:
+    name: Automation checks
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - name: Install chectl dependencies
+        run: yarn
+      - name: Pack chectl
+        run: yarn pack
```

Comment:
use specifically `yarn oclif-dev readme`

Reply:
We don't have such a command

yarn run v1.22.10
error Command "oclif-dev" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.